ios/Classes/SwiftOcarinaPlugin.swift

```diff
@@ -25,6 +25,13 @@ class LoopPlayer: Player {
         playerLooper = AVPlayerLooper(player: player, templateItem: playerItem)
         
         player.volume = Float(volume)
+        
+        do {
+            try AVAudioSession.sharedInstance().setCategory(AVAudioSession.Category.ambient, options: AVAudioSession.CategoryOptions.mixWithOthers)
+            try AVAudioSession.sharedInstance().setActive(true)
+        } catch let error {
+            print(error)
```

Comment:
Can't we have a better handling here? Maybe throw a platform exception, this way the error will fail silently.

Reply:
Sure. Sounds good